recaf-ui/src/main/java/me/coley/recaf/util/JigsawUtil.java

```diff
@@ -5,6 +5,7 @@
 import sun.misc.Unsafe;
```

Comment:
Should we be migrating from `sun.misc.Unsafe` to `jdk.internal.misc.Unsafe`?

As per: https://github.com/openjdk/jdk/commit/3789983e89c9de252ef546a1b98a732a7d066650#diff-8245d7af65e923f2eb697601df955fac3964d4459fadb78cae419478d1b40191

Reply:
`sun.misc.Unsafe` is currently not locked up, so no, we shouldn't